### DISCUSSION TITLE
Why does it only appear on the first connection and disappear afterwards

### DISCUSSION THREAD

**Discussion by u33cy7pbwkhn** [phantom/docs#298] [Help & Support] [answered]
### Summary

Why does it only appear on the first connection and disappear afterwards? How to implement it where each connection asks which wallet to use

### Steps to Reproduce

![photo_2024-09-24_11-58-08](https://github.com/user-attachments/assets/143b3ed9-166d-4ad6-b08c-84ee1e9c8914)

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by u33cy7pbwkhn** [comment]
![photo_2024-09-24_12-02-00](https://github.com/user-attachments/assets/19b2b4d4-6a79-4865-ba14-7b8461a0a1ef)
I clearly chose the default app wallet to always inquire about the status, but it still prompts this

**Comment by adamdelphantom** [comment, marked as answer]
Hey @u33cy7pbwkhn, there is no call in the provider api to be able to trigger this.
